src/Orleans.sln

```diff
@@ -188,6 +188,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestInterfaces", "TestInter
 EndProject
 Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "TestFSharpInterfaces", "TestFSharpInterfaces\TestFSharpInterfaces.fsproj", "{2375D7D5-9B30-493F-9C2E-A6B2811A3C5A}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "TestInternalDtos", "TestInternalDtos\TestInternalDtos.csproj", "{D2C5F0E3-E9C3-46F5-82E3-26D221246F59}"
```

Comment:
@sergeybykov: I know this is annoying. The problem is that the issue only occurs when I use a DTO which is in a *separate* assembly, the DTO is marked as *serializable* and the DTO is in an assembly which *references* a type from the Orleans.dll. When the DTO is in the test grain interfaces assembly, the issue does not occur.

Maybe you find a way the get rid off the additional assemblies; currently I am not able to reconstruct the issue with at least the additional assembly which is referencing Orleans.dll.